--- a/util/Makemodule.am	2020-04-26 09:14:58.000000000 +0000
+++ b/util/Makemodule.am	2020-11-07 07:24:34.152501353 +0000
@@ -12,7 +12,6 @@
 
 bin_PROGRAMS += \
 	fbsetroot \
-	fluxbox-remote \
 	fluxbox-update_configs
 
 fbsetroot_SOURCES = \
@@ -46,9 +45,3 @@
 	libFbTk.a \
 	$(FRIBIDI_LIBS)
 
-fluxbox_remote_SOURCES = \
-	util/fluxbox-remote.cc
-fluxbox_remote_CPPFLAGS = \
-	$(AM_CPPFLAGS) \
-	-I$(fbtk_incdir)
-
--- a/Makefile.in	2020-11-07 07:25:34.167504215 +0000
+++ b/Makefile.in	2020-11-07 07:27:40.506510239 +0000
@@ -124,7 +124,7 @@
 build_triplet = @build@
 host_triplet = @host@
 bin_PROGRAMS = fluxbox$(EXEEXT) fbsetroot$(EXEEXT) \
-	fluxbox-remote$(EXEEXT) fluxbox-update_configs$(EXEEXT) \
+	 fluxbox-update_configs$(EXEEXT) \
 	fbrun$(EXEEXT)
 @SYSTRAY_SRC_TRUE@@TOOLBAR_SRC_TRUE@am__append_1 = \
 @SYSTRAY_SRC_TRUE@@TOOLBAR_SRC_TRUE@	src/SystemTray.cc \
@@ -444,8 +444,6 @@
 	$(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1)
-am_fluxbox_remote_OBJECTS =  \
-	util/fluxbox_remote-fluxbox-remote.$(OBJEXT)
 fluxbox_remote_OBJECTS = $(am_fluxbox_remote_OBJECTS)
 fluxbox_remote_LDADD = $(LDADD)
 fluxbox_remote_DEPENDENCIES = libFbTk.a src/defaults.$(OBJEXT)
@@ -675,7 +673,6 @@
 	src/tests/$(DEPDIR)/testStringUtil-StringUtiltest.Po \
 	src/tests/$(DEPDIR)/testTexture-texturetest.Po \
 	util/$(DEPDIR)/fbsetroot-fbsetroot.Po \
-	util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Po \
 	util/$(DEPDIR)/fluxbox_update_configs-fluxbox-update_configs.Po \
 	util/fbrun/$(DEPDIR)/fbrun-FbRun.Po \
 	util/fbrun/$(DEPDIR)/fbrun-main.Po
@@ -1016,11 +1013,11 @@
 	data/windowmenu $(style_DATA) doc/CODESTYLE doc/fbrun.1.in \
 	doc/fbsetbg.1.in doc/fbsetroot.1.in doc/fluxbox-apps.5.in \
 	doc/fluxbox-keys.5.in doc/fluxbox-menu.5.in \
-	doc/fluxbox-remote.1.in doc/fluxbox-style.5.in \
+	doc/fluxbox-style.5.in \
 	doc/fluxbox.1.in doc/startfluxbox.1.in util/fbsetbg
 CLEANFILES = $(PATHFILES) doc/fbrun.1 doc/fbsetbg.1 doc/fbsetroot.1 \
 	doc/fluxbox-apps.5 doc/fluxbox-keys.5 doc/fluxbox-menu.5 \
-	doc/fluxbox-remote.1 doc/fluxbox-style.5 doc/fluxbox.1 \
+	doc/fluxbox-style.5 doc/fluxbox.1 \
 	doc/startfluxbox.1
 MAINTAINERCLEANFILES = \
 	aclocal.m4 \
@@ -1225,7 +1222,6 @@
 	doc/fluxbox-apps.5 \
 	doc/fluxbox-keys.5 \
 	doc/fluxbox-menu.5 \
-	doc/fluxbox-remote.1 \
 	doc/fluxbox-style.5 \
 	doc/fluxbox.1 \
 	doc/startfluxbox.1
@@ -1722,9 +1718,6 @@
 	libFbTk.a \
 	$(FRIBIDI_LIBS)
 
-fluxbox_remote_SOURCES = \
-	util/fluxbox-remote.cc
-
 fluxbox_remote_CPPFLAGS = \
 	$(AM_CPPFLAGS) \
 	-I$(fbtk_incdir)
@@ -2154,12 +2147,7 @@
 fluxbox$(EXEEXT): $(fluxbox_OBJECTS) $(fluxbox_DEPENDENCIES) $(EXTRA_fluxbox_DEPENDENCIES) 
 	@rm -f fluxbox$(EXEEXT)
 	$(AM_V_CXXLD)$(CXXLINK) $(fluxbox_OBJECTS) $(fluxbox_LDADD) $(LIBS)
-util/fluxbox_remote-fluxbox-remote.$(OBJEXT): util/$(am__dirstamp) \
-	util/$(DEPDIR)/$(am__dirstamp)
 
-fluxbox-remote$(EXEEXT): $(fluxbox_remote_OBJECTS) $(fluxbox_remote_DEPENDENCIES) $(EXTRA_fluxbox_remote_DEPENDENCIES) 
-	@rm -f fluxbox-remote$(EXEEXT)
-	$(AM_V_CXXLD)$(CXXLINK) $(fluxbox_remote_OBJECTS) $(fluxbox_remote_LDADD) $(LIBS)
 src/fluxbox_update_configs-defaults.$(OBJEXT): src/$(am__dirstamp) \
 	src/$(DEPDIR)/$(am__dirstamp)
 src/fluxbox_update_configs-Resources.$(OBJEXT): src/$(am__dirstamp) \
@@ -2402,7 +2390,6 @@
 @AMDEP_TRUE@@am__include@ @am__quote@src/tests/$(DEPDIR)/testStringUtil-StringUtiltest.Po@am__quote@ # am--include-marker
 @AMDEP_TRUE@@am__include@ @am__quote@src/tests/$(DEPDIR)/testTexture-texturetest.Po@am__quote@ # am--include-marker
 @AMDEP_TRUE@@am__include@ @am__quote@util/$(DEPDIR)/fbsetroot-fbsetroot.Po@am__quote@ # am--include-marker
-@AMDEP_TRUE@@am__include@ @am__quote@util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Po@am__quote@ # am--include-marker
 @AMDEP_TRUE@@am__include@ @am__quote@util/$(DEPDIR)/fluxbox_update_configs-fluxbox-update_configs.Po@am__quote@ # am--include-marker
 @AMDEP_TRUE@@am__include@ @am__quote@util/fbrun/$(DEPDIR)/fbrun-FbRun.Po@am__quote@ # am--include-marker
 @AMDEP_TRUE@@am__include@ @am__quote@util/fbrun/$(DEPDIR)/fbrun-main.Po@am__quote@ # am--include-marker
@@ -4249,20 +4236,6 @@
 @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
 @am__fastdepCXX_FALSE@	$(AM_V_CXX@am__nodep@)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(fluxbox_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) -c -o src/fluxbox-SystemTray.obj `if test -f 'src/SystemTray.cc'; then $(CYGPATH_W) 'src/SystemTray.cc'; else $(CYGPATH_W) '$(srcdir)/src/SystemTray.cc'; fi`
 
-util/fluxbox_remote-fluxbox-remote.o: util/fluxbox-remote.cc
-@am__fastdepCXX_TRUE@	$(AM_V_CXX)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(fluxbox_remote_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) -MT util/fluxbox_remote-fluxbox-remote.o -MD -MP -MF util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Tpo -c -o util/fluxbox_remote-fluxbox-remote.o `test -f 'util/fluxbox-remote.cc' || echo '$(srcdir)/'`util/fluxbox-remote.cc
-@am__fastdepCXX_TRUE@	$(AM_V_at)$(am__mv) util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Tpo util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	$(AM_V_CXX)source='util/fluxbox-remote.cc' object='util/fluxbox_remote-fluxbox-remote.o' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(AM_V_CXX@am__nodep@)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(fluxbox_remote_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) -c -o util/fluxbox_remote-fluxbox-remote.o `test -f 'util/fluxbox-remote.cc' || echo '$(srcdir)/'`util/fluxbox-remote.cc
-
-util/fluxbox_remote-fluxbox-remote.obj: util/fluxbox-remote.cc
-@am__fastdepCXX_TRUE@	$(AM_V_CXX)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(fluxbox_remote_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) -MT util/fluxbox_remote-fluxbox-remote.obj -MD -MP -MF util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Tpo -c -o util/fluxbox_remote-fluxbox-remote.obj `if test -f 'util/fluxbox-remote.cc'; then $(CYGPATH_W) 'util/fluxbox-remote.cc'; else $(CYGPATH_W) '$(srcdir)/util/fluxbox-remote.cc'; fi`
-@am__fastdepCXX_TRUE@	$(AM_V_at)$(am__mv) util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Tpo util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Po
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	$(AM_V_CXX)source='util/fluxbox-remote.cc' object='util/fluxbox_remote-fluxbox-remote.obj' libtool=no @AMDEPBACKSLASH@
-@AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
-@am__fastdepCXX_FALSE@	$(AM_V_CXX@am__nodep@)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(fluxbox_remote_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) -c -o util/fluxbox_remote-fluxbox-remote.obj `if test -f 'util/fluxbox-remote.cc'; then $(CYGPATH_W) 'util/fluxbox-remote.cc'; else $(CYGPATH_W) '$(srcdir)/util/fluxbox-remote.cc'; fi`
-
 src/fluxbox_update_configs-defaults.o: src/defaults.cc
 @am__fastdepCXX_TRUE@	$(AM_V_CXX)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(fluxbox_update_configs_CPPFLAGS) $(CPPFLAGS) $(AM_CXXFLAGS) $(CXXFLAGS) -MT src/fluxbox_update_configs-defaults.o -MD -MP -MF src/$(DEPDIR)/fluxbox_update_configs-defaults.Tpo -c -o src/fluxbox_update_configs-defaults.o `test -f 'src/defaults.cc' || echo '$(srcdir)/'`src/defaults.cc
 @am__fastdepCXX_TRUE@	$(AM_V_at)$(am__mv) src/$(DEPDIR)/fluxbox_update_configs-defaults.Tpo src/$(DEPDIR)/fluxbox_update_configs-defaults.Po
@@ -5362,7 +5335,6 @@
 	-rm -f src/tests/$(DEPDIR)/testStringUtil-StringUtiltest.Po
 	-rm -f src/tests/$(DEPDIR)/testTexture-texturetest.Po
 	-rm -f util/$(DEPDIR)/fbsetroot-fbsetroot.Po
-	-rm -f util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Po
 	-rm -f util/$(DEPDIR)/fluxbox_update_configs-fluxbox-update_configs.Po
 	-rm -f util/fbrun/$(DEPDIR)/fbrun-FbRun.Po
 	-rm -f util/fbrun/$(DEPDIR)/fbrun-main.Po
@@ -5564,7 +5536,6 @@
 	-rm -f src/tests/$(DEPDIR)/testStringUtil-StringUtiltest.Po
 	-rm -f src/tests/$(DEPDIR)/testTexture-texturetest.Po
 	-rm -f util/$(DEPDIR)/fbsetroot-fbsetroot.Po
-	-rm -f util/$(DEPDIR)/fluxbox_remote-fluxbox-remote.Po
 	-rm -f util/$(DEPDIR)/fluxbox_update_configs-fluxbox-update_configs.Po
 	-rm -f util/fbrun/$(DEPDIR)/fbrun-FbRun.Po
 	-rm -f util/fbrun/$(DEPDIR)/fbrun-main.Po
