--- a/browser/branding/branding-common.mozbuild
+++ b/browser/branding/branding-common.mozbuild
@@ -19,14 +19,18 @@
             'VisualElements_70.png',
         ]
         BRANDING_FILES += [
-            'bgstub.bmp',
+            'appname.bmp',
+            'bgintro.bmp',
             'branding.nsi',
+            'clock.bmp',
             'document.ico',
             'firefox.ico',
-            'firefox64.ico',
             'newtab.ico',
             'newwindow.ico',
+            'particles.bmp',
             'pbmode.ico',
+            'pencil-rtl.bmp',
+            'pencil.bmp',
             'wizHeader.bmp',
             'wizHeaderRTL.bmp',
             'wizWatermark.bmp',
@@ -41,22 +41,18 @@
         ]
     elif 'gtk' in CONFIG['MOZ_WIDGET_TOOLKIT']:
         BRANDING_FILES += [
-            'default128.png',
             'default16.png',
             'default22.png',
             'default24.png',
             'default256.png',
             'default32.png',
             'default48.png',
-            'default64.png',
         ]
         FINAL_TARGET_FILES.chrome.icons.default += [
-            'default128.png',
             'default16.png',
             'default22.png',
             'default24.png',
             'default256.png',
             'default32.png',
             'default48.png',
-            'default64.png',
         ]
