--- a/src/channel-webdav.c
+++ b/src/channel-webdav.c
@@ -363,6 +363,7 @@
     server = phodav_server_get_soup_server(spice_session_get_webdav_server(session));
 
     CHANNEL_DEBUG(self, "starting client %" G_GINT64_FORMAT, c->demux.client);
+    if(spice_session_get_shared_dir(session)!= NULL) {
 
     client = g_new0(Client, 1);
     client->refs = 1;
@@ -394,6 +395,7 @@
     g_clear_object(&peer);
     g_clear_error(&error);
     client_unref(client);
+   }
 #endif
 }
