--- a/src/bootstrap/builder.rs
+++ b/src/bootstrap/builder.rs
@@ -966,7 +966,7 @@
             let cflags = self.cflags(target).join(" ");
             cargo
                 .env(format!("CFLAGS_{}", target), cflags.clone())
-                .env("CFLAGS", cflags.clone());
+               .env("CFLAGS", cflags);
 
             if let Some(ar) = self.ar(target) {
                 let ranlib = format!("{} s", ar.display());
@@ -981,9 +981,7 @@
                 let cxx = ccacheify(&cxx);
                 cargo
                     .env(format!("CXX_{}", target), &cxx)
-                    .env("CXX", &cxx)
-                    .env(format!("CXXFLAGS_{}", target), cflags.clone())
-                    .env("CXXFLAGS", cflags);
+					.env("CXX", &cxx);
             }
         }
 
